_docs/platforms/kubernetes/examples/index.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I think this file and the entire platforms/kubernetes/examples/* tree can be git rm'd.